G/gRPC/build_tarballs.jl

```diff
@@ -0,0 +1,71 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "gRPC"
+version = v"1.48.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/grpc/grpc.git", "d2054ec6c6e8abcecf0e24b0b4ee75035d80c3cc")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/grpc && \
+git submodule update --init --recursive && \
+mkdir -p cmake/build && \
+pushd "cmake/build" && \
+cmake \
+    -DgRPC_INSTALL=ON \
+    -DgRPC_BUILD_TESTS=OFF \
+    -DCMAKE_INSTALL_PREFIX=${prefix} \
+    -DgRPC_SSL_PROVIDER=package \
+    ../.. && \
```

Comment:
This is not passing the cmake toolchain file?

Reply:
This configuration was done because of this: https://github.com/grpc/grpc/blob/master/BUILDING.md#cross-compiling 

and was lifted from here: https://github.com/grpc/grpc/blob/master/test/distrib/cpp/run_distrib_test_cmake_aarch64_cross.sh

Reply:
But we have [protoc](https://github.com/JuliaBinaryWrappers/protoc_jll.jl) already.  Also, the tarballs contains zlib, which it shouldn't, because again we have it already.

Reply:
Ahh, Zlib and protoc are fair, I've added those in. However, the build breaks when I add the toolchain flag to this bootstrap section.